js/dav/dav.js

```diff
@@ -1620,6 +1620,7 @@ var listAddressBooks = _co2['default'].wrap(regeneratorRuntime.mark(function cal
         req = request.propfind({
           props: [{ name: 'displayname', namespace: ns.DAV }, { name: 'owner', namespace: ns.DAV }, { name: 'getctag', namespace: ns.CALENDAR_SERVER }, { name: 'resourcetype', namespace: ns.DAV }, { name: 'sync-token', namespace: ns.DAV },
           //{ name: 'groups', namespace: ns.OC },
+          { name: 'read-only', namespace: ns.OC },
```

Comment:
It's the only line I changed on this file, the rest is just Github failing to see I didn't changed anything else.... 😑

Reply:
This is compiled code. I'm quite sure you shouldn't change it here, but in `js/dav/lib/contacts.js` and then run `make` in `js/dav`.

Reply:
Oh you're right! :O